velox/functions/sparksql/RegisterRand.h

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <string>
+
+namespace facebook::velox::functions::sparksql {
+void registerRandFunctions(const std::string& prefix);
```

Comment:
One more ask. Can you move this function to Rand.h and remove RegisterRand.h/cpp files?

Reply:
Yes, it makes sense to do that. Just moved in new commit. Also corrected the function namespace. Thanks!

Reply:
@PHILO-HE Thanks.